docs/installation/Installing_PKI_Server_with_Custom_NSS_Databases.md

```diff
@@ -0,0 +1,79 @@
+# Installing PKI Server with Custom NSS Databases
+
+## Overview
+
+This page describes the process to create a PKI server with custom NSS databases.
+
+Normally, when installing a PKI subsystem (e.g. CA) some NSS databases will be created by default, for example:
+* server NSS database: /etc/pki/pki-tomcat/alias
+* admin NSS database: ~/.dogtag/pki-tomcat/ca/alias
+
+Under some circumstances the admin may want to use custom NSS databases (e.g. with trust policy).
+In those cases the installation can be done in multiple steps:
+* create a basic PKI server
+* optionally, create a custom NSS database for the server
+* optionally, create a custom NSS database for the admin
+* install PKI subsystem with regular installation procedure
+
+## Creating Basic PKI Server
+
+To create a basic PKI server, execute the following command:
+
+```
+$ pki-server create
+```
+
+This will create a server in /var/lib/pki/pki-tomcat with configuration files in /etc/pki/pki-tomcat.
+
+See also [PKI Server CLI](https://www.dogtagpki.org/wiki/PKI_Server_CLI).
+
+## Creating Custom NSS Database for PKI Server
+
+To create a custom NSS database for the server execute the following commands:
+
+```
+$ pki-server nss-create --password <server password>
+```
+
+To enable trust policy:
+
+```
+$ modutil \
+    -dbdir /etc/pki/pki-tomcat/alias \
+    -add p11-kit-trust \
+    -libfile /usr/share/pki/lib/p11-kit-trust.so
+```
+
+See also [PKI Server NSS CLI](https://www.dogtagpki.org/wiki/PKI_Server_NSS_CLI).
+
+## Creating Custom NSS Database for PKI Administrator
+
+To create a custom NSS database for the admin execute the following commands:
+
+```
+$ pki -d ~/.dogtag/pki-tomcat/ca/alias -c <client password> nss-create
+```
+
+To enable trust policy:
```

Comment:
We can wordsmith this at a later date.

AI: @cipherboy - clarify what this does.